constrants/Headerconstraint.js

```diff
@@ -0,0 +1,40 @@
+import ICONS from "../components/headericons";
+import ROUTES from "../routes/routes";
+import headerTheme from "../Theme/HeaderTheme";
+
+const CONSTANTS = {
+  navbarPages: [
```

Comment:
This can be defined outside of the component in the file where you would need it or create a ROUTES variable in the constants folder which you can export from there to use wherever you need it